linera-views/src/backends/dual.rs

```diff
@@ -199,6 +199,9 @@ where
     const MAX_VALUE_SIZE: usize = usize::MAX;
 
     async fn write_batch(&self, batch: Batch) -> Result<(), Self::Error> {
+        if batch.is_empty() {
+            return Ok(());
+        }
```

Comment:
I wouldn't add this one and keep `dual.rs` unopinionated (e.g. concretely doesn't affect metrics).

Reply:
In general, how often are batches empty?

Reply:
Quite often when I did some test some time ago. But not recently.
Ok, for `dual.rs`.